app/views/board/editor.scala

```diff
@@ -21,7 +21,7 @@ object editor {
     moreJs = frag(
       jsAt(s"compiled/lichess.editor${isProd ?? (".min")}.js"),
       embedJsUnsafe(s"""var data=${safeJsonValue(bits.jsData(sit, fen, animationDuration))};data.positions=$positionsJson;
-LichessEditor(document.getElementById('board-editor'), data);""")
+LichessEditor.default(document.querySelector('#board-editor > .board-editor'), data);""")
```

Comment:
Looks like snabbdom replaces the element instead of replacing its content. I wonder if there is a more elegant way.

Reply:
I think in the other modules I just recreate the `main#board-editor` root with snabbdom.

Reply:
It's a bit awkward, because the editor can also be embedded in the study chapter dialog.